obspy/core/util/testing.py

```diff
@@ -203,10 +203,21 @@ def compare_images(expected, actual, tol):
             "shape %s." % (str(actual_image.shape),
                            str(expected_image.shape)))
 
+    if actual_image.shape[2] == 4:
+        has_alpha_channel = True
```

Comment:
Do we need to keep this alpha channel handling? If it is not going to be used it might become stale.

Reply:
Yeah, guess you're right, I'll remove it.